docs/basic_usage.rst

```diff
@@ -64,7 +64,7 @@ You can also get the current ``FiscalQuarter`` with:
 FiscalDateTime
 --------------
 
-The start and end of each quarter are stored as instances of the ``FiscalDateTime`` class. This class provides all of the same features as the ``datetime`` class, with the addition of the ability to query the fiscal year and quarter.
+The start and end of each quarter are stored as instances of the ``FiscalDateTime`` class. This class provides all of the same features as the ``datetime`` class, with the addition of the ability to query the fiscal year, fiscal month, and quarter.
```

Comment:
Can you also make these same changes to the README?

Reply:
These review comments are a bit confusing. Not sure if this is just the same comment from before but I believe I did?

https://github.com/adamjstewart/fiscalyear/pull/11/commits/b8bc20506b1f208f7793b5d3221459a4e051ab25

Reply:
Different comment on a different file. README contains the initial test of index.rst and the full text of basics.rst.